### DISCUSSION TITLE
Set Up - Failed Tasks - data seeder seed

### DISCUSSION THREAD

**Discussion by cleverclope** [opencrvs/opencrvs-core#7359] [Technical Q&A]
### OpenCRVS Core version

1.5.0

### OpenCRVS Countryconfig version

1.5.0

### Your operating system and version

Ubuntu 22.04

### Your node version

18.

### Which of these describes your project best?

- [X] Testing OpenCRVS for business development, educational purposes or for personal interest.
- [ ] A research project to evaluate, configure and test OpenCRVS as a possible solution for a government, an active brief or proposal.
- [ ] Actively installing and configuring OpenCRVS Core for production use in a country.
- [ ] Customising OpenCRVS Core code in production in a country

**Comment by Zangetsu101** [comment]
Hi @cleverclope, thanks for trying out OpenCRVS. Could you perhaps provide us with more information about the failure? For example, any error output shown in the console.
